ui_framework/doc_site/src/actions/code_viewer_actions.js

```diff
@@ -0,0 +1,12 @@
+import ActionTypes from './action_types';
+
+export default {
```

Comment:
This is very "un-ES6-y". This should rather be:

```js
export openCodeViewer: source => ({
  // ...
})

export closeCodeViewer: () => ({
  // ...
})
```

Then you can `import { openCodeViewer } from '...'` instead. This also makes tree shaking and other things possible (not necessarily relevant for this use case, I just think an ES6 module should almost never default export an object that just contains functions either way).